dtsync/sync.go

```diff
@@ -96,6 +96,7 @@ func (s *Sync) Close() error {
 		log.Warnf("Closing datatransfer sync with %d syncs in progress", len(s.syncDoneChans))
 	}
 	for _, ch := range s.syncDoneChans {
+		ch <- errors.New("sync closed")
```

Comment:
Should this be done in a `select` in case there are no readers, or do we want to wait for the channel to be read?

Reply:
We don’t want a caller to think a sync succeeded when it didn’t. And anyways this is buffered.

Reply:
Probably safe since the channel is buffered